firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -27,39 +27,28 @@ void moveMotor(int motor[], int steps, int dir)
 
 void moveSpaces(int motor[], int spaces, int dir)
 {
-  int i;
-
   // How many steps per space
   float numSteps = spaces * stepsPerSpace;
```

Comment:
Now that I'm thinking about it, this should be an int. We're not doing half of a step or anything.